sway/commands.c

```diff
@@ -169,6 +170,72 @@ static bool cmd_exit(struct sway_config *config, int argc, char **argv) {
 	return true;
 }
 
+static bool cmd_floating(struct sway_config *config, int argc, char **argv) {
+	if (strcasecmp(argv[0], "toggle") == 0) {
```

Comment:
Based on the i3 user guide, I don't think that there are any other commands for floating, but I left this open ended in case we wanted to extend functionality or if there was something I didn't see.